child_compassion/wizard/project_description_fr.py

```diff
@@ -13,6 +13,14 @@
 class Project_description_fr:
 
     @classmethod
+    def _get_translated_value(cls, value):
```

Comment:
Cannot this _get_translated_value be put in classe translated_value so that the same code is written only once for all languages on both child descriptions and project descriptions ?